code/game/objects/structures/barsigns.dm

```diff
@@ -111,6 +111,11 @@
 	icon = "slipperyshots"
 	desc = "Slippery slope to drunkeness with our shots!"
 
+/datum/barsign/thegreytide
+	name = "The Grey Tide"
+	icon = "thegreytide"
+	desc = "Take a break from toolboxing and enjoy a lazy beer!"
```

Comment:
I'd prefer that. "Take a break" implies that you're gonna have a beer and charge right back out.